### PR TITLE
[Merged by Bors] - chore: no need for universe polymorphism in convexHull_eq

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by semorrison** [comment]
This is motivated by the porting note at https://github.com/leanprover-community/mathlib4/blob/master/Mathlib/Analysis/Convex/Jensen.lean#L133.

File: Mathlib/Analysis/Convex/Combination.lean [hunk 1]
```diff
@@ -297,10 +297,11 @@ theorem convexHull_range_eq_exists_affineCombination (v : ι → E) : convexHull
     exact affineCombination_mem_convexHull hw₀ hw₁
 #align convex_hull_range_eq_exists_affine_combination convexHull_range_eq_exists_affineCombination
 
-/-- Convex hull of `s` is equal to the set of all centers of masses of `Finset`s `t`, `z '' t ⊆ s`.
-This version allows finsets in any type in any universe. -/
+/--
+Convex hull of `s` is equal to the set of all centers of masses of `Finset`s `t`, `z '' t ⊆ s`.
+-/
```
Comment (ADedecker) [inline Mathlib/Analysis/Convex/Combination.lean:302]:
```suggestion
For universe reasons, you shouldn't use this lemma to prove that a given center of mass belongs 
to the convex hull. Use convexity of the convex hull instead.
-/
```

I don't feel strongly about this, so feel free to ignore if you think it's useless, but I could imagine people trying to use this lemma to do everything and having to fight universes issues.

**Comment by ADedecker** [comment]
bors d+

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**